roles/github/templates/build-kayobe-docker-image.yml.j2

```diff
@@ -9,6 +14,11 @@ env:
 
 jobs:
   %% format_file_name(workflow.file_name) %%:
+    <%- if github_environment_type == 'kayobe' +%>
+    strategy:
+      matrix:
+        environment: %% github_kayobe_environments %%
```

Comment:
Do we need a separate Kayobe image for each environment?

Reply:
The concern is related to multiple environments being configured in such a way that a runner in one environment couldn't reach the Pulp server within another. Therefore easier to assume this is case and build and push separate images to all environments.

Reply:
I suppose we need to know what will be the most common approach. I general though I'd prefer we use the same image everywhere, as we do for the kolla images.

Reply:
Could add a boolean to allow users to decide if they want separate builds for all environments or one build to multiple.